packages/calypso-e2e/src/lib/pages/themes-page.ts

```diff
@@ -86,13 +93,21 @@ export class ThemesPage extends BaseContainer {
 		}
 
 		// Select the first available theme as the target.
-		const selectedTheme = await this.page.waitForSelector( `:nth-match(${ selector }, 1)` );
-
-		// Hover over the target theme to expose the `INFO` button and wait for the animation to
-		// complete.
-		await selectedTheme.hover();
-		await selectedTheme.waitForElementState( 'stable' );
+		return await this.page.waitForSelector( `:nth-match(${ selector }, 1)` );
+	}
 
-		await Promise.all( [ this.page.waitForNavigation(), selectedTheme.click() ] );
+	/**
+	 * Given a target theme and action, click on the popover item of the action on the theme.
+	 *
+	 * @param {ElementHandle} selectedTheme Reference to the target theme.
+	 * @param {string} action Action to be called from the popover.
+	 */
+	async clickPopoverItem(
+		selectedTheme: ElementHandle,
```

Comment:
I've decided on having this method accept an ElementHandle as a parameter instead of calling `this.select` within the method for the following reasons:

- less ambiguity: it is clearer that this method will be called on a specific theme card. 
- decouples `select` and `action` logic: even if the `select` call is modified again in the future it should not be necessary to modify all methods that depend on `select`.